src/components/NcUserBubble/NcUserBubble.vue

```diff
@@ -239,8 +263,23 @@ export default {
 			return this.url && this.url.trim() !== ''
 		},
 
+		route() {
+			return getRoute(this.$router, this.url)
+		},
```

Comment:
Let's add explicit `to` param similar to other components, so we can provide route location, not only URL string.

Do we have any place where we cannot explicitly set `url`/`to` and we need auto router integration?

Reply:
I would also prefer the explicit `to` prop instead of auto route.